### PR TITLE
trpl: mention missing_docs lint

### PR DESCRIPTION
**Pull request by panicbit**

### CODE REVIEW

File: src/doc/trpl/documentation.md [hunk 1]
```diff
@@ -546,6 +546,31 @@ extern crate foo;
 pub use foo::bar;
 ```
 
+## Missing documentation
+
+Sometimes you want to make sure that every single thing in your project is
```
Comment (killercup) [inline src/doc/trpl/documentation.md]: "every single _public_ thing"?

**Comment by killercup** [comment]
Nice, I'm using this all the time.

Might be good place to mention `#[doc(hidden)]` as well.

**Comment by panicbit** [comment]
@killercup What's the difference between `#[doc(hidden)]` and normal comments?

**Comment by killercup** [comment]
It hides elements that would otherwise be included by rustdoc.
panicbit <notifications@github.com> schrieb am Fr., 9. Okt. 2015 um 13:59:

**Comment by panicbit** [comment]
@killercup Thanks!

**Comment by panicbit** [comment]
Reopened to restart the travis build. It looks like it just failed to download some files.

**Comment by steveklabnik** [comment]
Thanks!

**Pull request merged**